sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.ContentSafety.netstandard2.0.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.AI.ContentSafety
+{
+    public partial class ContentSafetyClient
+    {
+        protected ContentSafetyClient() { }
+        public ContentSafetyClient(string endpoint, Azure.Core.TokenCredential credential, Azure.AI.ClientOptions options = null) { }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
+    }
```

Comment:
This still shows no methods. Did you run _eng/scripts/CodeChecks.ps1 contentsafety_ and commit?

Reply:
Yep I've already ran eng/scripts/CodeChecks.ps

Reply:
Hi @heaths, do you have any updated suggestion for this problem? I've created an issue reporting this bug: https://github.com/Azure/autorest.csharp/issues/3342. We plan to release this sdk in //build, and target merge this PR by 5/12, would appreciate if you could help pushing the fix of this issue.

Reply:
This should have all public members in https://github.com/Azure/azure-sdk-for-net/pull/34933/files#diff-5ba963678ef26de570a5282ed29b4864b1ecdb8a94084d4f74a50f5050c97305 and all your public models defined in this file. Does the script show any errors when you run it? Does this project even build?

As far as releasing, you're still missing a README, samples, and more - much of which is required to release. We have validations in place that make sure your README has appropriate content, as well as CHANGELOGs. The APIs also have to be reviewed, and if this file isn't getting populated there's nothing for the architects to review and approve.

Reply:
Hi @heaths, thanks for the response! I have uploaded tests and samples, and will work on the README and CHANGELOG before PR get merged.
This SDK has already been reviewed by SDK architects, please refer to this [link](https://apiview.dev/Assemblies/Review/d1fee5210fa34184bca6003f087171e3).
![image](https://user-images.githubusercontent.com/50131889/236097456-be64a89d-9577-43e2-9664-0e27246dcfcb.png)
We've already got approval for the first release. Ted said for the first release only the approval of package name is needed, and we will collect customer feedbacks during the Public Preview.
I'm wondering if we could have a quick sync this week about the remaining items before package release. And maybe I can show you the problem I met during code generation then.

Reply:
You need to fix those for builds to work. Looks like your _.sln_ file references a non-existent project - probably just the wrong path. You can edit it by hand. You should be able to run `dotnet build` or even `dotnet test` in your solution directly cleanly.

@m-nash I know some edits were made after the code was generated, but I wonder if the template needs to be fixed. Perhaps the _.sln_ doesn't have the right template like it once did.

Reply:
The _stress/*.csproj_ project is missing. You should open a tracking issue to add it back, but for now I can remove it from the _.sln_.

Reply:
That's not the problem I'm seeing. When I checked out and build your PR, there are compilation errors e.g.,

> error CS0246: T
he type or namespace name 'IUtf8JsonSerializable' could not be found (are you missing a using directive or an assembly reference?) [C:\src\az
ure-sdk-for-net\sdk\contentsafety\Azure.AI.ContentSafety\src\Azure.AI.ContentSafety.csproj::TargetFramework=netstandard2.0]

@m-nash this is a generated client, but what are they missing that this isn't compiling?

Also, since there are models shouldn't be there client teaspoons? I don't see those in this PR.

Reply:
@heaths I've tried some mitigation of this issue: https://github.com/Azure/autorest.csharp/issues/3341, and after this mitigated, the operations seem generated successfully. I pushed it here in another branch: https://github.com/mengaims/azure-sdk-for-net/blob/carnegie-test/sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.ContentSafety.netstandard2.0.cs
I'm confirming with Crystal whether my mitigation is valid, if so, I think we could replace with latest generated code.

Btw, I once encountered the same build error of IUtf8JsonSerializable, but after I close the solution and reopen again it disappeared, you could give it a try.

Reply:
Why did you push it to another branch? This PR should've been updated. We can't merge it until it's in a buildable state.

Reply:
You get models with tsp without a client.tsp they just use all default settings.  Client.tsp is only needed for customization.  The reason this `IUtf8JsonSerializable` error is happenig is because there is no tsp-location.yaml file checked in with this PR that I could see.